.github/workflows/ruby_edge.yml

```diff
@@ -0,0 +1,69 @@
+name: Edge Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+  schedule:
+    - cron: "10 4 * * */2"
+
+jobs:
+  edge-test:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_JOBS: 4
+      BUNDLE_RETRY: 3
+      BUNDLE_GEMFILE: gemfiles/graphql_master.gemfile
+      DATABASE_URL: postgres://postgres:postgres@localhost:5432
+      ELASTICSEARCH_URL: http://localhost:9200
```

Comment:
Why do we need these two? The setup must be similar to `ruby.yml`, the only difference is the Gemfile.

And I'm not sure we need to extract this into a separate workflow; it can be another parameter in the `ruby.yml` matrix.

Also, `ruby_edge` is a confusing name; we're not testing against edge Ruby but graphql gem.

Reply:
Because in the matrix we put "head" ruby (for master ruby), and this ruby version puts `RUBY_VERSION` to the docker file through env (it is default, like ci cd looks for ruby.yml).
So Docker doesn't know about the "head" ruby version.
Also, I couldn't find the image with the master ruby

About naming - OK, agree